units/commit_queue.go

```diff
@@ -156,6 +160,10 @@ func (j *commitQueueJob) Run(ctx context.Context) {
 		j.AddError(errors.Wrapf(err, "can't find commit queue for id %s", j.QueueID))
 		return
 	}
+	if cq == nil {
```

Comment:
I don't think this will ever be true (because of the way [this](https://github.com/evergreen-ci/evergreen/blob/bffee4b933f245118a2d211b77d5dfa60787ff61/model/commitqueue/db.go#L48) was written)

Reply:
Oh well in that case, I think that Find function should be changed: Sam and I had a conversation about this so we could rope him in but at this point in our code base the convention is to have Find functions return nil error for nil items.

Reply:
sgtm

Reply:
don't forget:
https://github.com/evergreen-ci/evergreen/blob/b42ea499b084481bce0d0ce075ee33e81bb2062b/rest/data/project.go#L122

Reply:
Thanks!